.gitignore

```diff
@@ -1,2 +1,3 @@
 book
 stable-check/
+.idea
```

Comment:
You can add this to `~/.config/git/ignore` to apply it for your whole system or `.git/info/exclude` to apply it to this repo without having to commit it.